doc/source/data/loading-data.rst

```diff
@@ -507,7 +507,7 @@ Ray Data interoperates with distributed data processing frameworks like `Daft <h
     .. tab-item:: Dask
 
         To create a :class:`~ray.data.dataset.Dataset` from a
-        `Dask DataFrame <https://docs.dask.org/en/stable/dataframe.html>`__, call
+        `Dask DataFrame <https://docs.getdaft.io/en/stable/api/dataframe/>`__, call
```

Comment:
This is not correct, right? It updates from dask to daft (the original link seems to still be active)

Reply:
Good catch! Sorry, I figured these links had already been validated by the submitter.

What's the policy here for validating link updates?

Reply:
I don't think we have a policy, we just need to check the PRs manually